src/pages/CarsList/CarsList.scss

```diff
@@ -13,12 +15,28 @@
     color: blueviolet;
   }
 }
-
 .mb-2.btn-group {
-  display: block;
+  display: flex;
   margin-top: 2%;
   margin-left: 2%;
+  padding: 0.25rem;
+  width: 40%;
+  background-color: $lightGrey;
   button:focus {
-    background-color: black;
+    background-color: $black;
+    color: $white;
+    box-shadow: none;
   }
-}
+  button:hover {
+    background-color: $black;
+    color: $white;
+    box-shadow: none;
+  }
+  button {
+    background-color: $lightGrey;
+    color:  $black;
+    border-radius: 0%;
+    margin: 1%;
```

Comment:
`%` ???
border-radius can be 0
and I don't see in the figma designs that its given 1% margin